quantum/encoder.c

```diff
@@ -81,12 +81,33 @@ void encoder_init(void) {
     }
 #endif
 
-    for (int i = 0; i < NUMBER_OF_ENCODERS; i++) {
-        setPinInputHigh(encoders_pad_a[i]);
-        setPinInputHigh(encoders_pad_b[i]);
+#ifdef ENCODER_PINS
```

Comment:
Naming things is difficult, however i'm cautious of too much crossover to existing setting, eg `BACKLIGHT_PINS`. While docs exists, it leads to questions like "why doesn't `#define ENCODER_PINS {A1, B2}` work?"

I cant find an existing setting to align to, but the closest thing that come to mind would be `ENCODER_PIN_MODE`, maybe `ENCODER_PAD_MODE`.